README.md

```diff
@@ -4,6 +4,32 @@ This module implements a parser for the `Contextual Query Language` (CQL) which
 `Search/Retrieval via URL` (SRU) and `Search/Retrieval via Web Service` (SRW) family of protocols.
 See the [Library of Congress spec](https://www.loc.gov/standards/sru/cql/) for details.
 
+## Usage
+
+Running `make` or `go build -o . ./...` will compile the library and create the command-line tool `cql-cli`.
```

Comment:
Do you need `.` as well as `./...`?

Reply:
The first dot is value for -o which puts the binary in current directory.

Reply:
Oh, how stupid of me. Yes, of course.